### DISCUSSION TITLE
Android App Links

### DISCUSSION THREAD

**Discussion by bmachek** [immich-app/immich#10195] [General]
Hi,

would it be possible to use App Links in the Immich app? 
It would be great if users who receive an album invitation where redirected instantly to the Immich mobile app instead of the browser, once they click on an album link. 

Documentation links:
https://developer.android.com/studio/write/app-link-indexing
https://docs.flutter.dev/cookbook/navigation/set-up-app-links
https://docs.flutter.dev/cookbook/navigation/set-up-universal-links

Best regards,
Bastian

**Comment by bo0tzz** [comment]
#2881

**Comment by Pozi09** [comment]
Please tell us more about your app
